tools/orca/src/profile.rs

```diff
@@ -45,6 +45,16 @@ impl Profile {
         self.idm_admin_password.as_str()
     }
 
+    pub fn set_admin_password(&mut self, pw: &str) {
+        self.admin_password.clear();
+        self.admin_password.push_str(pw);
+    }
+
+    pub fn set_idm_admin_password(&mut self, pw: &str) {
+        self.idm_admin_password.clear();
+        self.idm_admin_password.push_str(pw);
```

Comment:
some weird efficiency paranoia :p Idk I felt it was better to update the string content rather than allocate a new one? As if it mattered since it literally does it two times tho